frontend/scss/photoView.scss

```diff
@@ -94,12 +94,88 @@ rect {
     padding-left: 10px;
 }
 
+.side-button {
+    font-size: 18px;
+    font-weight: 500;
+    border: 2px solid;
+    padding: 2px 10px;
+    background-color: white;
+}
+
+.side-button:hover {
+    background-color: #20CCD7;
+}
+
+.tag-button {
+    font-size: 16px;
+    border: 2px solid #FB2F2A;
+    padding: 2px 24px;
+    margin: 5px;
+    background-color: white;
+}
+
+.tag-button:hover {
+    background-color: #f6afae;
+}
+
+.photo-sort-dropdown-button {
+    font-size: 16px;
+    border-width: 0px;
+    border-bottom: 2px solid black;
+    padding: 0px 22px 0px 0px;
+    margin: 5px;
+    border-radius: 0px;
+    background-color: white;
+    color: #5E5F5F;
+}
+
+.photo-sort-dropdown-menu {
+    font-size: 16px;
+    border-width: 0px;
+    padding: 2px 24px;
+    margin: 5px;
+    border-radius: 15px;
+    background-color: #00B1DB;
+    color: white;
+}
+
 .analysis-overlay {
     position: absolute;
     top: 0;
     left: 0;
 }
 
-.image-box{
+.image-box {
     position: relative;
+    padding: 25px;
+    background-color: black;
+}
+
+.similar-photo {
+    max-width: 100px;
```

Comment:
Since we want these to be responsive, let's use em values that will change depending on screen size